### PR TITLE
Added --force-bgz option to Importvcf to use BGzip codec to read .gz

### PR DESCRIPTION
**Pull request by lfrancioli**

### CODE REVIEW

File: src/main/java/org/broadinstitute/hail/io/compress/BGZipCodecGZ.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.hail.io.compress;
+
+public class BGzipCodecGZ extends BGzipCodec {
```
Comment (danking) [inline src/main/java/org/broadinstitute/hail/io/compress/BGZipCodecGZ.java]: @lfrancioli this needs a capital `Z` in `zip`.

**Comment by tpoterba** [comment]
Laurent, I was totally wrong about being able to do this per-command -- I'm really sorry.  I thought that it would be possible to create a new configuration just for this command and use that, but this is only possible for `HadoopConfiguration`s and not `SparkContext`s.  Can you reopen the old PR?  That model is our only option.

**Comment by lfrancioli** [comment]
Hi Tim,

What's the problem with this implementation? I've tested it and it works...

**Comment by tpoterba** [comment]
This sets the configuration permanently -- any following commands will use the overridden codecs.  Setting a global option is almost certainly better than getting this kind of leakage, I think

**Comment by lfrancioli** [comment]
No no, I reset the codecs afterwards. I tested and it works as intended
(loading a .gz annotation file with the Gzip codec). I'm trying to fix the
small letter / capital issue (thanks Daniel), but it Git seems to be
case-insensitive when it comes to files...

**Comment by lfrancioli** [comment]
OK, I fixed the case issue.

**Comment by tpoterba** [comment]
Oh, great!  I didn't notice that you were setting the conf back at the end.  Perfect.

**Pull request merged**